testFile.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const {  transports } = require('winston');
+const { transports } = require('winston');
 
```

Comment:
if u want to actually test it you should reassign stdout and check that the logs are written to there

Reply:
good point 👍

Reply:
fixed